### PR TITLE
Benchmarking

### PR DESCRIPTION
**Pull request by Alek99**
### All Submissions:

- [ ] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/reflex-dev/reflex/blob/main/CONTRIBUTING.md) file?
- [ ] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/reflex-dev/reflex/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### New Feature Submission:

- [ ] Does your submission pass the tests? 
- [ ] Have you linted your code locally prior to submission?

### Changes To Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.

    b. Describe your changes.

    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: .github/workflows/codespaces.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: codspeed-benchmarks
```
Comment (masenf) [inline .github/workflows/codespaces.yml:1]: feels like this file should be named `.github/workflows/codspeed.yml`?

File: .github/workflows/codespaces.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    # Note even though this job is called "pre-commit" and runs "pre-commit", this job will run
+    # also POST-commit on main also!  In case there are mishandled merge conflicts / bad auto-resolves
+    # when merging into main branch.
```
Comment (masenf) [inline .github/workflows/codespaces.yml:9]: not sure i get the context here

**Review by masenf — COMMENTED** [review]
ah, still draft 😃

**Comment by Alek99** [comment]
Yeah going to clean it up more just testing it

File: .github/workflows/codespaces.yml (hunk 1 above)
Comment (Alek99) [reply .github/workflows/codespaces.yml]: True going to change it

**Pull request closed without merging**